### PR TITLE
[MM-29403] migrate string-refs in custom_theme_chooser

### PR DESCRIPTION
**Pull request by filipghorbani**
#### Summary

Replace this.refs to using React.createRef in custom_theme_chooser.jsx

#### Ticket Link

Fixes https://github.com/mattermost/mattermost-server/issues/15842
JIRA ticket: https://mattermost.atlassian.net/browse/MM-29403

#### Related Pull Requests

#5753

### CODE REVIEW

File: components/user_settings/display/user_settings_theme/custom_theme_chooser.jsx [hunk 1]
```diff
@@ -129,6 +129,11 @@ export default class CustomThemeChooser extends React.PureComponent {
         this.state = {
             copyTheme,
         };
+
+        this.textarea = React.createRef();
```
Comment (Willyfrog) [inline components/user_settings/display/user_settings_theme/custom_theme_chooser.jsx:133]: In order to have consistency with other PRs, can we rename the reference? Can you do the same with the other refs?
```suggestion
        this.textareaRef = React.createRef();
```

**Review by Willyfrog — CHANGES_REQUESTED** [review]
hi there! thanks for contributing
added a comment for naming consistency
also I didn't see the references being assigned to a dom object, you will have to do that in places like in https://github.com/mattermost/mattermost-webapp/blob/b012a3a9cdbc1ae0bc04c819156fbdbbb3c2b6fc/components/user_settings/display/user_settings_theme/custom_theme_chooser.jsx#L336
take a look at the reference PR if you need.

**Review by sudheerDev — APPROVED** [review]
LGTM, Thank you @filipghorbani

**Comment by sudheerDev** [comment]
@filipghorbani Can you also remove `/* eslint-disable react/no-string-refs */` instances from the file

**Comment by Willyfrog** [comment]
@filipghorbani sorry it took me a while to come back to your PR, looks good but it is now conflicting with another PR removing jquery. can you update this to fix the conflict?

**Review by Willyfrog — CHANGES_REQUESTED** [review]
Hi! it is conflicting with the jquery migration that is happening at the same time, could you update this PR so we don't loose the jquery change? (I added a comment previously but forgot to mark it as a change request)

**Comment by Willyfrog** [comment]
@filipghorbani let me know if I can help you with the conflict somehow! I'm happy to help

**Comment by Willyfrog** [comment]
closing for inactivity, feel free to ask to work on it again and thanks for the efforts :)

**Pull request closed without merging**